### PR TITLE
Release Documentation - 1.5.0

### PR DESCRIPTION
**Pull request by kris-nova**
# Changelog

We will need to make sure this PR contains the changelog updates for the release

# Release process

We should go through the release process when merging this PR

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1126)

### CODE REVIEW

**Comment by chrislovecnm** [comment]
Maybe we have a doc under our dev folder for a HOWTO release.  See https://github.com/kubernetes/release/blob/master/README.md ... we are going to be a lot simpler :)

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 1.4.2
+
+* API Machinery
+* Private Network Topologies in AWS
+* Deprecating Model YAML manifests
```
Comment (justinsb) [inline HISTORY.md]: I don't think we even need to mention this... it's just for developers.

Comment (kris-nova) [inline HISTORY.md]: Pulled out

**Comment by kris-nova** [comment]
@chrislovecnm because you added comments on the PR, and not in the code here are my responses

> We need to move HISTORY.md to CHANGELOG.md

Done, and I also nuked `CHANGES.md`

> Maybe we have a doc under our dev folder for a HOWTO release.

You mean like [this?](https://github.com/kubernetes/kops/blob/master/docs/releases.md#release-process)

**Comment by kris-nova** [comment]
Also - just to be clear on expectations in this PR..

Please god - do **NOT** merge this until we are all 3 ready to make the release. This PR serves as the PR documented [here](https://github.com/kubernetes/kops/blob/master/docs/releases.md#2-merge-the-release-notes) in our release process. After this is merged, we **MUST** do things like create binaries, etc..

**Comment by chrislovecnm** [comment]
@kris-nova do we need this still?  I assume yes

**Comment by kris-nova** [comment]
@chrislovecnm - yes let me update for 1.5.0 and fix the conflicts

**Comment by chrislovecnm** [comment]
Marked this as WIP, so that we hopefully do not merge :)

**Pull request closed without merging**